content/cockpit-146.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cockpit 146
+author: Martin Pitt
+date: 2017-07-21 10:00
+tags: cockpit, linux
+slug: cockpit-146
+category: release
+summary: Cockpit with Software Updates improvements and GCE support
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 146.
+
+### Software Updates page shows more information
+
+The "Available Updates" and "Restart recommended" pages now show the packages
```

Comment:
In previous blogs we never quoted the Page Titles at all, and referred to UI elements with ``. Updated accordingly. Thanks for spotting!